assets/js/components/notifications/dashboard-core-site-alerts.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * DashboardCoreSiteAlerts component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import Notification from 'GoogleComponents/notifications/notification';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getNotifications,
+	acceptNotification,
+	dismissNotification,
+} from './site';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+class DashboardCoreSiteAlerts extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			notifications: false,
+		};
+	}
+
+	async componentDidMount() {
+		const notifications = await getNotifications().catch( ( e ) => {
+			// eslint-disable-next-line no-console
+			console.warn( 'Error caught while fetching notifications', JSON.stringify( e ) );
```

Comment:
I often use `global.console` for these. It probably doesn't matter? I dunno!

Reply:
Also: why stringify the error? `console` can output objects.

Reply:
Yeah, I know but it doesn't work when logging from E2E tests.

https://travis-ci.com/google/site-kit-wp/jobs/291425794#L1104

I didn't mean to keep it like that though so I'll update it.